unit2.js

```diff
@@ -0,0 +1,94 @@
+// function getOption() {
+//     var obj = document.getElementById("mySelect");
+//     document.getElementById("demo").innerHTML = 
+//     obj.options[obj.selectedIndex].text;
+//   }
+
+
+const getFilms = () => {
+    const filmUrl = "https://ghibliapi.herokuapp.com/films"
+    fetch(filmUrl)
+        .then(response => response.json())
+        .then(response => {
+            //console.log(response)
+            getFilmTitle(response);
+        })
+        .catch(error => {
+            console.log("Error! Something is not working!")
+        })
+
+}
+
+getFilms()
+
+const getFilmTitle = (response) => {
+    let filmArray = [];
+    for (let i = 0; i < response.length; i++) {
+        let titles = response[i].title
+        filmArray.push(titles)
+    }
+    sendTitles(filmArray)
+
+    console.log(filmArray)
+}
+
+const myParent = document.querySelector("#container-for-select-box")
+const selectList = document.getElementById("mySelect");
+
+selectList.id = "mySelect";
+const sendTitles = (filmArray) => {
+    for (let i = 0; i < filmArray.length; i++) {
+        let option = document.createElement("option");
+        option.text = filmArray[i];
+        selectList.appendChild(option);
+        //selectList.addEventListener("change", getValues)
+        //getValues(response)
+    }
+}
+
+function getOption() {
+    let selectBox = document.getElementById("mySelect");
+    document.getElementById("demo").innerHTML = 
```

Comment:
This is a creative way of getting which movie is selected from options. Another way of doing this would be to use event listener with event.target attribute.